### PR TITLE
PXC-3789: PXC 8.0.27 refresh - Q4 2021 (wsrep-lib)

### PR DESCRIPTION
**Pull request by kamil-holubicki**
https://jira.percona.com/browse/PXC-3789

Merged upstream commit 
edd141127c11d78ef073f9f3ca61708821f20b32

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "wsrep-API/v26"]
 	path = wsrep-API/v26
-	url = https://github.com/percona/wsrep-API.git
-	branch = percona-4.x-8.0
+	url = https://github.com/kamil-holubicki/wsrep-API.git
+	branch = PXC-3789
```
Comment (satya-bodapati) [inline .gitmodules]: This should be avoided?

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
@kamil-holubicki are there two more commits after the merge commit?

File: .gitmodules (hunk 1 above)
Comment (kamil-holubicki) [reply .gitmodules]: Yes @satya-bodapati . This is to allow Jenkins to build the project. Once wsrep-API is approved and merged. I will change this to point to the official repo.
Please continue your review and let me know if any other doubts.

**Review by satya-bodapati — APPROVED** [review]

File: .gitmodules (hunk 1 above)
Comment (satya-bodapati) [reply .gitmodules]: ok, go ahead then

**Comment by satya-bodapati** [comment]
> @kamil-holubicki are there two more commits after the merge commit?

as discussed on slack, we will bring them in next merge

**Review by dutow — APPROVED** [review]

**Review by venkatesh-prasad-v — APPROVED** [review]

**Pull request merged**